app/_gateway_entities/plugin.md

```diff
@@ -42,6 +43,73 @@ The set of plugins you have access to depends on your license tier.
 
 You can also develop custom plugins, adding your own custom functionality to {{site.base_gateway}}.
 
+## Plugin precedence
```

Comment:
I'd move this section to plugin scopes. I had questions about precedence while I was reading it and this is more detailed reference material than the intro/overview content that's on the entity pages.

Reply:
Oh, i like this idea. I struggled with keeping precedence info as a separate page, but if it goes with scopes, it fits more naturally.